sql/plan/process.go

```diff
@@ -185,11 +227,22 @@ func (t *ProcessTable) PartitionRows(ctx *sql.Context, p sql.Partition) (sql.Row
 	return &trackedRowIter{iter: iter, onNext: onNext, onDone: onDone}, nil
 }
 
+type queryType byte
+
+const (
+	queryTypeSelect = iota
```

Comment:
Is it okay that `CALL`, `SIGNAL`, `LOAD DATA`, etc. get slapped under `queryTypeSelect`?

Reply:
Signal is 0, good catch.

Reply:
Not sure about CALL, left a TODO